### ISSUE TITLE
Perhaps "locking" some threads on the Wall

### ISSUE THREAD

**Issue by RyckRichards** [Tatoeba/tatoeba2#1963]
As all of you know, there were long, intensives debates and also. Decisions have to be made.  (Maybe we call they announcements.)

Some debates were (kinda) valid but it would be a waste of time to keep that threat "alive" (allowing other users to debate) since it was clear that nothing "new" or productive would emerge. 

For announcements, maybe they would be not "that" accepted and some members would tend to create more and more replies that would make it difficult to check the new announcements. Maybe the announcement would be something like "We need ... If you'd like to apply, leave us a private message" or even "User abcde is applying to be a ***. Leave your feedback by sending a private message (Maybe "abcde" would not want feedback on public.)) 

Perhaps in these cases we could manage to allow only admins to reply (maybe add an information ) by "locking them" (not allowing another users to reply). 

Content posted by other users can be deleted but we could avoid doing so by not allowing (of course, in certain cases) them to reply to a threat.

**Comment by jiru** [comment]
If a thread is locked, members will just react in a separate thread instead. By implementing such a locking mechanism, do you want to force members to react in a separate thread, or do you want to prevent them from reacting at all? If the former: why? If the latter: I understand that you guys went through a lot lately, but I don't think censorship will help in any way.

**Comment by ckjpn** [comment]
> If a thread is locked, members will just react in a separate thread instead.

This is true, so locking wouldn't stop the discussion.

If the problem is that it's hard to follow long discussion on the Wall, perhaps a forum format rather than the Wall would be useful. 

Related:
https://github.com/Tatoeba/tatoeba2/issues/367
(It's a closed issue.)

**Comment by trang** [comment]
Closing this.

We do have cases of threads that go a bit out of control, but those cases are rare and having a lock mechanism will indeed not solve much for us. Sometimes we need to have difficult discussions and it's okay that people get angry or emotional about a certain topic.

We already have mechanisms to put a stop at threads that we can't handle anymore: we can hide posts as a signal that the conversation is becoming toxic and if that's not enough, we can suspend users temporarily so that they take the time to calm down.
